rotkehlchen/history/events.py

```diff
@@ -25,8 +25,8 @@
 log = RotkehlchenLogsAdapter(logger)
 
 
-# eth transactions, external trades, ledger actions, uniswap trades, makerDAO DSR,
-# makerDAO vaults, yearn vaults, compound, adex staking, aave lending
+# eth transactions, external trades, ledger actions, balancer trades, uniswap trades,
```

Comment:
what's this comment for? I don't even remember

Reply:
It was there :S I just extended it adding balancer. I'll remove it.